editors/intellij/src/main/kotlin/com/github/biomejs/intellijbiome/settings/BiomeSettingsConfigurable.kt

```diff
@@ -0,0 +1,35 @@
+package com.github.biomejs.intellijbiome.settings
+
+import com.github.biomejs.intellijbiome.BiomeBundle
+import com.github.biomejs.intellijbiome.services.BiomeServerService
+import com.intellij.openapi.components.service
+import com.intellij.openapi.options.BoundSearchableConfigurable
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.ui.DialogPanel
+import com.intellij.openapi.vfs.VirtualFile
+import com.intellij.ui.dsl.builder.bindText
+import com.intellij.ui.dsl.builder.panel
+
+class BiomeSettingsConfigurable(internal val project: Project): BoundSearchableConfigurable(BiomeBundle.message("biome.settings.name"), BiomeBundle.message("biome.settings.name")) {
+    override fun createPanel(): DialogPanel {
+        val settings: BiomeSettings = BiomeSettings.getInstance(project)
+			val biomeServerService = project.service<BiomeServerService>()
+
+			return panel {
+            row {
+                textFieldWithBrowseButton(BiomeBundle.message("biome.path.label")) { fileChosen(it) }
+                    .bindText(settings::executablePath)
+            }
```

Comment:
I think we need a label for this input. I didn't understand what to set until I read the code.

<img width="1078" alt="SS_ 2023-09-18 11 51 15" src="https://github.com/biomejs/biome/assets/17228098/057c1567-d250-4b76-83b2-1463d41efd7f">